_plugins/helixKramdown.rb

```diff
@@ -0,0 +1,45 @@
+# modeled after https://github.com/mvdbos/kramdown-with-pygments/blob/master/kramdown_pygments.rb
+require 'kramdown/converter'
+
+# Custom plugin for Kramdown. It's used by the custom converter, not by Jekyll.
+module Kramdown
+  module Converter
+    class HelixHtml < Html
+      def convert_header(el, indent)
+        attr = el.attr.dup
+        if @options[:auto_ids] && !attr['id']
+          attr['id'] = generate_id(el.options[:raw_text])
+        end
+        @toc << [el.options[:level], attr['id'], el.children] if attr['id'] && in_toc?(el)
+        level = output_header_level(el.options[:level])
+        if level == 1
+          attr['class'] = 'hxPageTitle'
+        elsif level == 2
+          attr['class'] = 'hxSectionTitle'
+        elsif level == 3
+          attr['class'] = 'hxSubSectionTitle'
+        else
```

Comment:
Add the case for level 4 header `hxContainerTitle`.  I've seen a couple articles with the need for sub sub titles.

Aside from that this is really, really awesome!

:shipit:

Reply:
done :)